prettier.config.js

```diff
@@ -0,0 +1,7 @@
+
+// prettier.config.js or .prettierrc.js
+module.exports = {
+  tabWidth: 2,
+  semi: true,
+  singleQuote: true,
```

Comment:
Personally, don't care that much about the formatting rules. (e.g. single quote vs double)But, I care that we use the same one. (this is just a copy/paste of the prettier docs)